cmd/init.go

```diff
@@ -59,57 +66,62 @@ func Init() *cobra.Command {
 				return err
 			}
 
-			if err := executeInit(devPath, overwrite, l, workDir); err != nil {
+			if err := executeInit(namespace, devPath, l, workDir, overwrite); err != nil {
 				return err
 			}
 
-			log.Success(fmt.Sprintf("Okteto manifest (%s) created", devPath))
+			log.Success(fmt.Sprintf("Okteto manifest (%s) created. Check its content in case you need to adapt it", devPath))
```

Comment:
```suggestion
			log.Success(fmt.Sprintf("Okteto manifest (%s) created", devPath))
```

That second sentence sounds bad, let's not have it.